.github/workflows/upload_new_tag_to_pypi.yml

```
@@ -0,0 +1,58 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      # with:
+      #   python-version: 3.9
+
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user

```

```suggestion
      run: >-
        python -m
        pip install
        build
        twine
        --user

```
Why not install build and twine at the same time instead of in separate steps?